[2024-08-09T09:29:28.007+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-09T09:29:28.037+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Integrate_Data_Warehouse.print_start manual__2024-08-09T09:29:21.245407+00:00 [queued]>
[2024-08-09T09:29:28.049+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Integrate_Data_Warehouse.print_start manual__2024-08-09T09:29:21.245407+00:00 [queued]>
[2024-08-09T09:29:28.050+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-09T09:29:28.086+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): print_start> on 2024-08-09 09:29:21.245407+00:00
[2024-08-09T09:29:28.091+0000] {standard_task_runner.py:63} INFO - Started process 4985 to run task
[2024-08-09T09:29:28.094+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Integrate_Data_Warehouse', 'print_start', 'manual__2024-08-09T09:29:21.245407+00:00', '--job-id', '428', '--raw', '--subdir', 'DAGS_FOLDER/etl_ldtbxh.dag.py', '--cfg-path', '/tmp/tmpuet0rhqz']
[2024-08-09T09:29:28.097+0000] {standard_task_runner.py:91} INFO - Job 428: Subtask print_start
[2024-08-09T09:29:28.186+0000] {task_command.py:426} INFO - Running <TaskInstance: Integrate_Data_Warehouse.print_start manual__2024-08-09T09:29:21.245407+00:00 [running]> on host 45f7764b2bf2
[2024-08-09T09:29:28.338+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Integrate_Data_Warehouse' AIRFLOW_CTX_TASK_ID='print_start' AIRFLOW_CTX_EXECUTION_DATE='2024-08-09T09:29:21.245407+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-09T09:29:21.245407+00:00'
[2024-08-09T09:29:28.339+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-09T09:29:28.355+0000] {logging_mixin.py:188} INFO - Starting to Integrate Data Warehouse...
[2024-08-09T09:29:28.356+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-09T09:29:28.357+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-09T09:29:28.370+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Integrate_Data_Warehouse, task_id=print_start, run_id=manual__2024-08-09T09:29:21.245407+00:00, execution_date=20240809T092921, start_date=20240809T092928, end_date=20240809T092928
[2024-08-09T09:29:28.431+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-09T09:29:28.481+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-09T09:29:28.494+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
